app/connectors/httpParsers/ResponseHttpParsers.scala

```diff
@@ -33,8 +35,10 @@ trait ResponseHttpParsers {
   private def generateClientError(error: ApiError): Left[HttpError, Nothing] = {
     error match {
       case ApiSingleError(code, message) => Left(BadRequestError(code, message))
-      case ApiMultiError(_, _, _) => Left(MultipleErrors)
-      case ApiMultiErrorFinancial(_) => Left(MultipleErrors)
+      case ApiMultiError(code, message, errorResponse) =>
```

Comment:
IntelliJ warning here as _code_ and _message_ in this case are not being used, so can be replaced with underscores.